_includes/google_analytics.html

```diff
@@ -1,3 +1,4 @@
+<!-- Initialize analytics for the TTS Google Analytics account -->
```

Comment:
Wondering if this would be better as a Liquid comment, so that it's not included in the HTML shipped to users?

```suggestion
{% comment %}
Initialize analytics for the TTS Google Analytics account
{% endcomment %}
```

Then again, it's not entirely irrelevant to end-users to include the comment in the markup, since it could provide some insight / relief into where we're directing analytics data.

Fine either way.

Reply:
I had a similar thought! but I think it's clearer in the HTML because that's the first place people check, before going to the template

Reply:
We're including a public comment in the DAP analytics block, so I figured this one deserved a public comment as well.